Packs/Claroty/Integrations/Claroty/Claroty.yml

```diff
@@ -0,0 +1,312 @@
+category: Authentication
+commonfields:
+  id: Claroty
+  version: -1
+configuration:
+- display: CTD Server URL (e.g. https://<IP>:5000)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
```

Comment:
please place the `proxy` and `insecure` params above the fetch incidents related params